### ISSUE TITLE
Better support for Galaxy Fold

### ISSUE THREAD

**Issue by HackerChick** [dslul/openboard#835]
Editing per great suggesrion by @StormyIceLeopard.

**Is your feature request related to a problem? Please describe.**
Tl;Dr; OpenBoard's keyboard layouts / sizing and ability to adapt between open (double width) and closed/cover (narrow) views make it difficult and frequently outright unusable on the Fold

Details: The keyboard is much too wide for gesture typing when the phone is opened up. It's also so tall that it takes up a disproportionate amount of screen real estate to that its hard to see the screen's content. Scaling down the height helps, but a reasonable height (eg 50-60%) for the opened phone is now disproportionately too wide, and then becomes much too small to use when the phone is closed (on the narrow cover screen). Similarly the "one hand" mode it's still much too wide to easily gesture type with your thumb. Finally, and I hesitate to write this as I realize it is quite subjective, but personally I find the Split Keyboard layout to be unusable (too difficult/unnatural to use) on the Open Fold and so not a viable option. 

**Describe the solution you'd like**
Implement a split keyboard using the key layout / spacing / sizing of GBoard's split keyboard, and have it then switch to a normal/appropriately sized keyboard when the phone is closed. This seems to be the layout that works best (by a very long shot to anything I've found) on the Fold.

And then, provide a thumb option for gesture typing that provides the correct proportions when the Fold is open (and returns to the normal keyboard when closed). Not to be a total GBoard fanboy/girl, but I would again recommend using GBoard's sizing/spacing for this thumb keyboard as it feels quite natural. 

**Describe alternatives you've considered**
Ability to scale both the width and the height for all of the different options (normal, split, thumb) with different settings for when phone is open (or in landscape) vs when is closed. This strikes me as quite problematic/much less flexible to new form factors and sizes that might appear. So I don't think this is a good option.

**Comment by StormyIceLeopard** [comment]
Also something to consider

Split keyboard when open
Normal when closed

Kind of does this already
But not consistent unfortunately

**Comment by Koruudo** [comment]
+1

**Comment by HackerChick** [comment]
Yes!!! To split keyboard. Except split keyboard with the layout/spacing that GBoard uses (which I only discovered after posting this and am now convinced it is far and away the most workable keyboard I've seen on the Fold. I'm trying OpenBoard's split layout but finding it super tough to type with on the Fold).

I'm not sure the right etiquette,but I'm going to edit my solution in my 1st post to this. I would REALLY like to get back to openboard and off of google!! Thanks, @StormyIceLeopard.

**Comment by StormyIceLeopard** [comment]
May want to move here.
A fork is under active development

https://github.com/openboard-team/openboard/issues/828
